rita_common/src/debt_keeper/mod.rs

```diff
@@ -602,12 +616,17 @@ impl DebtKeeper {
                     return Ok(DebtAction::OpenTunnel);
                 }
 
-                info!(
-                    "debt {} is below close threshold {} for {}. suspending forwarding",
-                    debt_data.debt, close_threshold, ident.wg_public_key
-                );
-                debt_data.action = DebtAction::SuspendTunnel;
-                Ok(DebtAction::SuspendTunnel)
+                if enable_enforcement {
+                    info!(
+                        "debt {} is below close threshold {} for {}. suspending forwarding",
+                        debt_data.debt, close_threshold, ident.wg_public_key
+                    );
+                    debt_data.action = DebtAction::SuspendTunnel;
+                    Ok(DebtAction::SuspendTunnel)
+                } else {
+                    debt_data.action = DebtAction::OpenTunnel;
+                    Ok(DebtAction::OpenTunnel)
+                }
```

Comment:
This is where we use enable enforcement. On subsequent ticks, it should return OpenTunnel instead of SuuspendTunnel which would undo any active enforcement?

Reply:
yes I believe that will do it.